cli/src/pcluster/cli.py

```diff
@@ -308,6 +316,28 @@ def _get_parser():
     _addarg_region(pami)
     pami.set_defaults(func=build_image)
 
+    # delete image command subparser
+    pdeleteami = subparsers.add_parser(
+        "delete-image",
+        help="Deletes an image.",
```

Comment:
"Deletes an image and related image builder stack".?

Reply:
Done